README.md

```diff
@@ -39,6 +41,51 @@ anonymize(data);
 // }
 ```
 
+#### Example using serializers
+
+```js
+const { anonymizer } = require('@uphold/anonymizer');
+const whitelist = ['foo.key', 'foo.depth.*', 'bar.*', 'toAnonymize.baz'];
+const blacklist = ['foo.depth.innerBlacklist'];
+const serializers = [
```

Comment:
Where are the serializers being used? Shouldn't be an option when instantiating the anonymizer?

Reply:
Done